hub/router/routes.go

```diff
@@ -224,7 +216,7 @@ func UpdateUser(c *gin.Context) {
 		return
 	}
 
-	if user == (models.User{}) {
+	if user.Username == "" || user.Password == "" || (user.Role == "user" && len(user.WorkspaceIDs) == 0) {
```

Comment:
We want to update users without mandatorily supplying a password in the UI, right now if you try to update a user and you do not add a password value, update will fail

Reply:
It does make a lot of sense. I will adjust it...

Reply:
Done in [commit](https://github.com/shamanec/GADS/pull/174/commits/60171a9866d44d9e4171d88eae61717a4ad86d1c)